netket/operator/_kinetic.py

```diff
@@ -102,5 +104,11 @@ def _expect_kernel(
     def _pack_arguments(self) -> PyTree:
```

Comment:
this is really ugly, I hope at some point we can modify it... in fact what is called and stored and mass is not the mass, it's the inverse of the mass...!